bip-0039/portuguese.txt

```diff
@@ -0,0 +1,2048 @@
+esmola
+esnobe
+espanto
+espelho
+espiga
+esporo
+espreita
+esquadro
+esquerda
+esqui
+estaca
+esterco
+estima
+estofado
+estrela
+etanol
+etiqueta
+euforia
+eunuco
+evasivo
+eventual
+exagero
+exame
+exato
+exemplo
+expresso
+externo
+extinto
+extrato
+fabril
+fabuloso
+facada
+faceta
+facial
+fadiga
+fagulha
+faixa
+fajuto
+falido
+falta
+familiar
+fantoche
+faquir
+fardado
+farelo
+farinha
+farofa
+farrapo
+fartura
+fasor
+fatia
+fator
+fauno
+fava
+favela
+faxina
+fazenda
+febre
+fecho
+fedor
+feio
+feira
+feitio
+feixe
+felino
+feminino
+feno
+fera
+ferrolho
+ferrugem
+fervente
+festa
+feto
+feudal
+fiapo
+fibroso
+fidalgo
+figa
+figurado
+fileira
+filho
+filme
+findo
+fineza
+finitude
+finura
+firmeza
+fissura
+fita
+fivela
+fixo
+flagelo
+flanela
+flerte
+floco
+flora
+fluido
+fluxo
+focinho
+foco
+fogo
+foguete
+foice
+folgado
+folheto
+folia
+fome
+fonema
+fonte
+forca
+forense
+formado
+forno
+forte
+fosco
+fossa
+fractal
+frade
+fragata
+fralda
+frango
+fraque
+frasco
+fraterno
+freio
+frente
+fresta
+frevo
+friagem
+friso
+fritura
+fronte
+frota
+frouxo
+fugaz
+fulano
+fuligem
+fungo
+funil
+futebol
+fuzil
+gabarito
+gado
+gafe
+gagueira
+gaiato
+gaiola
+gaivota
+galante
+galeria
+galho
+galinha
+galocha
+ganho
+ganido
+garfo
+gargalo
+garimpo
+garoto
+garrafa
+garrote
+gasto
+gatuno
+gaveta
+gazeta
+geada
+geleia
+gelo
+gemada
+gemido
+generoso
+gengiva
+genitor
+genoma
+genro
+gerador
+gergelim
+geriatra
+germinal
+gesso
+gestor
+giga
+ginasta
+gincana
+gingado
+girafa
+girino
+glacial
+glicose
+global
+gnose
+goela
+goiaba
+goleiro
+golfe
+gomo
+gongo
+gordura
+gorjeta
+gorro
+gosma
+gosto
+goteira
+gracejo
+gracioso
+gradual
+grafite
+gralha
+gramado
+granada
+grato
+graveto
+graxa
+grego
+greve
+grife
+grilo
+gritaria
+grosso
+grudado
+gruta
+guarda
+gueixa
+guelra
+gueto
+guia
+guilda
+guincho
+guisado
+guizo
+gula
+gume
+guru
+gutural
+harpia
+haste
+havana
+hectare
+hediondo
+hera
+heresia
+hiato
+hibisco
+hidrovia
+hiena
+hindu
+hino
+hipismo
+hipnose
+hipoteca
+histeria
+hoje
+homem
+homicida
+honesto
+honrado
+horda
+horrendo
+horto
+humorado
+iate
+idade
+ideia
+idem
+idoso
+iene
+ignitor
+ignorado
+igreja
+iguana
+ileso
+ilha
+ilibado
+iludido
+ilustre
+imagem
+imantado
+imaturo
+imenso
+imersivo
+iminente
+imitante
+imortal
+impacto
+imperial
+implante
+imposto
+imprensa
+impune
+imune
+inalador
+inapto
+incauto
+incenso
+inciso
+incolor
+inculto
+indeciso
+indireto
+indolor
+indutor
+ineficaz
+inepto
+inercial
+inexato
+infarto
+inferno
+infinito
+influxo
+informal
+infrator
+ingrato
+inhame
+inibido
+inicial
+inimigo
+injetor
+injusto
+inocente
+inodoro
+inovador
+inquieto
+insulto
+integral
+intruso
+inundado
+invasor
+inverno
+invicto
+invocado
+iodo
+ioga
+iogurte
+irado
+ironia
+irreal
+irritado
+isca
+isento
+isolado
+isqueiro
+jacente
+jacobino
+jacu
+jade
+jaleco
+jamais
+jambo
+janela
+jangada
+jantar
+japona
+jaqueta
+jararaca
+jardim
+jarro
+jasmim
+jato
+javali
+jazida
+jeca
+jegue
+jeito
+jejum
+jenipapo
```

Comment:
Another example https://www.infopedia.pt/dicionarios/lingua-portuguesa/jenipapo.

Reply:
Since it's a fruit and there's no other name in Portugal I think it's ok to leave it there.
But if it's too strange for someone from Portugal we could think about replacing, maybe as low priority. The only thing is that it is now becoming hard to find good words that don't match previous word lists from other languages.

Also, another rule I didn't explicitly wrote (because it's not strictly followed) is that I tried to write mainly concrete nouns so people could use even photos to describe their seed. I think the less abstract the better. It was very hard to come up with this whole list, so maybe we won't be able to follow this rule for further replacements. But let's do our best.